Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Prefabs/HandMenuSliderAssets/Scripts/UpdateTrackScale.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+
+
+public class UpdateTrackScale : MonoBehaviour
```

Comment:
this class still needs to be in a namespace.

Reply:
Ended up removing this script as it was redundant. Thanks!